packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -222,7 +222,7 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     if (minVisitStopDatetime && visitStopDatetime <= minVisitStopDatetime) {
       validSubmission = false;
       setError('visitStopDate', {
-        message: t('invalidVisitStopDate', 'Stop date needs to be on or after {{lastEncounterDatetime}}', {
+        message: t('invalidVisitStopDateAccEncounter', 'Stop date needs to be on or after {{lastEncounterDatetime}}', {
```

Comment:
I don't understand the `AccEncounter` part here; I get we need to disambiguate `invalidVisitStopDate`, but `visitStopDateMustBeAfterMostRecentEncounter` seems better. Translation keys are translator-facing and should be named in such a way that they facilitate understanding the use of this key.

Reply:
Just bad choice of words on my side.
Thanks for the improvement. I'll take care of the same from the next time.